pkg/apicapi/apic_sync.go

```diff
@@ -38,6 +38,15 @@ func (conn *ApicConnection) apicBodyAttrCmp(class string,
 		bodyd.Attributes = make(map[string]interface{})
 	}
 	for p, def := range meta.attributes {
+		if class == "vzRsSubjGraphAtt" && p == "tnVnsAbsGraphName" {
+			_, forceResolveExists := bodyc.Attributes["forceResolve"]
```

Comment:
As I understand, this allows the user to change the graph associated with the contract. So the user has to necessarily create a new service graph with the FW inserted? Should we not have an exclusion case for vnsAbsGraph as well..

Reply:
> So the user has to necessarily create a new service graph with the FW inserted?

Yes.

Reply:
vnsAbsGraph is created by user (Not editing the one created by cni ), so there will not be any aci cni annotations and we will not process it. But User edits the graphName of the contract created by aci cni, and since it has annotations, we we process it and we have to ignore the field while processing